core/pkg/server/sender.go

```diff
@@ -1058,9 +1059,9 @@ func (s *Sender) sendFile(file *service.FilesItem) {
 				s.fwdChan <- record
 			},
 		)
-		task.AddCompletionCallback(s.fileTransferManager.FileStreamCallback())
-		task.AddCompletionCallback(
+		task.SetCompletionCallback(
 			func(*filetransfer.Task) {
+				s.fileTransferManager.FileStreamCallback()
```

Comment:
This function seems to return a callback; should the return value be invoked?

```go
s.fileTransferManager.FileStreamCallback()()
```

though if that's the case, it's definitely a little error prone